### ISSUE TITLE
Reworked Podman API

### ISSUE THREAD

**Issue by ThraaxSession** [ThraaxSession/gintainer#19]
Currently the podman client ist just a wrapper, which uses the podman command.
For better accessibility, the Golang Bindings of podman should be used.

https://github.com/containers/podman/blob/main/pkg/bindings/README.md

Important:
* Check if the tests are working, if not rewrite the tests.
* Check if the APIs are still working.
* Check for improvements in the workflows
